lib/vm.js

```diff
@@ -17,27 +17,25 @@ const realRunInThisContext = Script.prototype.runInThisContext;
 const realRunInContext = Script.prototype.runInContext;
 
 Script.prototype.runInThisContext = function(options) {
-  if (options && options.breakOnSigint) {
-    return sigintHandlersWrap(() => {
-      return realRunInThisContext.call(this, options);
-    });
-  } else {
-    return realRunInThisContext.call(this, options);
-  }
+  let breakOnSigint = options && options.breakOnSigint;
+  let runInThisContext = () => realRunInThisContext.call(this, options);
+
+  if (breakOnSigint) return sigintHandlersWrap(runInThisContext);
+
+  return runInThisContext();
 };
 
 Script.prototype.runInContext = function(contextifiedSandbox, options) {
-  if (options && options.breakOnSigint) {
-    return sigintHandlersWrap(() => {
-      return realRunInContext.call(this, contextifiedSandbox, options);
-    });
-  } else {
-    return realRunInContext.call(this, contextifiedSandbox, options);
-  }
+  let breakOnSigint = options && options.breakOnSigint;
+  let runInContext = () => realRunInContext.call(this, contextifiedSandbox, options);
+
+  if (breakOnSigint) return sigintHandlersWrap(runInContext);
+
+  return runInContext();
 };
 
 Script.prototype.runInNewContext = function(sandbox, options) {
-  var context = exports.createContext(sandbox);
+  let context = exports.createContext(sandbox);
```

Comment:
it would be preferred if you din't rename these since we aren't touching them otherwise. :)